README.md

```diff
@@ -932,6 +933,19 @@ Provides the capability to add confetti with various types ( diamonds, star, mix
 <img src="https://raw.githubusercontent.com/Intuit/CardParts/master/images/confetti.gif" width="300" alt="Confetti"/>
 </p>
 
+### `CardPartProgressBarView`
+
+Provides the capability to configure different colors and custom marker , it's position to indicate the progress based ont he value provided.
```

Comment:
Spelling error.  `on the` and the comma has a space after `marker` ;)

Reply:
Fixed.